clusterloader2/testing/load/dnsperf-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+{{$HostNetworkMode := DefaultParam .CL2_USE_HOST_NETWORK_PODS false}}
+{{$CpuRequest := DefaultParam .CpuRequest "10m"}}
+{{$MemoryRequest := DefaultParam .MemoryRequest "10M"}}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+  labels:
+    group: load
+    svc: {{.SvcName}}-{{.Index}}
+spec:
+  replicas: {{RandIntRange .ReplicasMin .ReplicasMax}}
+  selector:
+    matchLabels:
+      name: {{.Name}}
+  template:
+    metadata:
+      labels:
+        group: load
+        name: {{.Name}}
+        svc: {{.SvcName}}-{{.Index}}
+    spec:
+      hostNetwork: {{$HostNetworkMode}}
+      containers:
+      # Fetches the dns server from /etc/resolv.conf and
+      # runs as 5 clients and 2 threads sending 10 queries per second.
+      # dnsperf has a client timeout of 5s. It sends queries for 60s,
+      # then sleeps for 10s, to mimic bursts of DNS queries.
+      - command:
+        - sh
+        - -c
+        - server=$(cat /etc/resolv.conf | grep nameserver | cut -d ' ' -f 2); echo
+          "Using nameserver ${server}"; while true; do ./dnsperf -s $server -l 60
+          -d /etc/queries/all-queries -c 5 -T 2 -Q 10; sleep 10; done
+        image: k8s.gcr.io/kube-dns-perf-client-amd64:1.1
+        imagePullPolicy: Always
```

Comment:
Is it really needed? I suggest changing imagePullPolicy to something that will reduce load on nodes.